photo-app/app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,59 @@
+const show_error = function (message) {
+  if(!$("#flash-messages").length){
```

Comment:
```
$("#flash-messages")
```
assign this to a variable so that won't keep hitting the DOM tree to fetch the value